### ISSUE TITLE
[Bug] Install-DbaFirstResponderKit installs deprecated versions of stored proc

### ISSUE THREAD

**Issue by alevyinroc** [sqlcollaborative/dbatools#6531]
Issue originally reported by @chambone in Slack; they or @alevyinroc will complete  the environmental information within 24 hours.

### Environmental information

```powershell
# Replace with output of Get-Culture
```

#### SQL Server:

```sql
/* REPLACE WITH output of @@VERSION */
```

```sql
/* REPLACE WITH output of @@LANGUAGE */
```

### Report

When installing Brent Ozar's First Responder Kit, the Deprecated/obsolete versions of `sp_Blitz`, `sp_BlitzCache` and `sp_BlitzIndex` are installed by overwriting the currently-supported versions.

Root cause: `Install-DbaFirstResponderKit` looks for [`sp_*.sql` recursively in the directory](https://github.com/sqlcollaborative/dbatools/blob/7c8e6417cdd03725e24f6ab8a8f6c559d767d554/functions/Install-DbaFirstResponderKit.ps1#L199) that the zip file is unpacked into. The "live" versions are installed first, then the Deprecated versions are installed.

Suggested resolution: Add a new switch to control whether deprecated versions of scripts are installed. Default to `$false`. If set to `$false`, do not use `-Recurse` with `Get-ChildItem`

#### Host used

- [X] powershell.exe
- [ ] ISE
- [ ] VS Code
- [ ] Other (please specify)

If anything other than powershell.exe was used, please confirm that you can duplicate the issue with powershell.exe

- [ ] Still buggy in powershell.exe

#### Steps to Reproduce

1. Run `Install-DbaFirstResponderKit`
2. Review version of these three stored procedures that are installed.

#### Expected Behavior

Current versions should be installed.
#### Actual Behavior
Deprecated versions are installed

**Comment by wsmelton** [comment]
@alevyinroc I went ahead and took care of this one because I was reworking that command.

I figured out the recurse was being done because it was creating a root directory under Dbatools data path and then dumping the archive under that directory. So it ended up creating `AppData\Roaming\PowerShell\SQL-Server-First-Responder-Kit-master\SQL-Server-First-Repsonder-Kit-master\...` path, and without doing the Recurse itself you would never find the SQL files.

I reworked how we pull the latest release and how the latest or dev zip file is extracted so it does not do the duplicate folder noted above.
